streampark-console/streampark-console-webapp/src/store/modules/locale.ts

```diff
@@ -36,6 +39,7 @@ export const useLocaleStore = defineStore({
     setLocaleInfo(info: Partial<LocaleSetting>) {
       this.localInfo = { ...this.localInfo, ...info };
       ls.set(LOCALE_KEY, this.localInfo);
+      cookies.set(LANGUAGE, this.localInfo?.locale || 'en', { path: '/' });
```

Comment:
I found frontend `localInfo` is differ from backend `Locale.SIMPLIFIED_CHINESE`

https://github.com/SbloodyS/incubator-streampark/blob/4788f1dab18964ac810238e4a389ce6f7509f568/streampark-console/streampark-console-webapp/types/config.d.ts#L13